bedrock/foundation/templates/foundation/base-resp.html

```diff
@@ -30,6 +30,7 @@
 {% block content %}
   <main>
     <header class="page-header">
+      {% block search %}search{% endblock %}
```

Comment:
Should leave the block empty here so "search" doesn't show up on other pages (e.g. `/foundation/advocacy/`).